libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts

```diff
@@ -674,7 +673,7 @@ describe('DropdownComponent', () => {
     });
 
     describe('when aligned to right side of viewport', () => {
-      it('should align the dropdown to the right side of button and component container ', (done) => {
+      xit('should align the dropdown to the right side of button and component container ', (done) => {
```

Comment:
We still have to address these flaky tests somehow.
I do not think we should merge this to develop until we have an indication of what is causing this sudden explosion in flaky tests - develop is soon to be released and it introduces too much uncertainty.

Reply:
done